tests/farmers.test.js

```diff
@@ -30,7 +30,7 @@ describe('Farmer route', () => {
   let id = '';
   it('Login user', (done) => {
     const userLogin = {
-      username: 'James',
+      username: 'james',
```

Comment:
I think this is causing the tests to fail, because user with `username: 'james'` will have different ID than user with `username: 'James'`.

Can you look into it please? 🙏